folly/ExceptionWrapper.h

```diff
@@ -20,7 +20,11 @@
 #pragma once
 
 #include <cassert>
+#if __has_include(<cstdint>) 
```

Comment:
What specifically is this needed for? Not having `cstdint` is a significant issue for compiling a large number of C++ programs, and I don't know if it's worth trying to support such a platform.

Reply:
Fixed by updating Dinkumware headers.